examples/methods/plot_kmm_reweighting.py

```diff
@@ -0,0 +1,65 @@
+"""
+Kernel Mean Matching
+====================
+This example illustrates the use of KMM method [1] to correct covariate-shift.
+.. [1] J. Huang, A. Gretton, K. Borgwardt, B. Schölkopf and A. J. Smola.
+       'Correcting sample selection bias by unlabeled data.'
+       In NIPS, 2007.
+"""
+
+# Author: Antoine de Mathelin
+#
+# License: BSD 3-Clause
+# sphinx_gallery_thumbnail_number = 1
+
+# %%
+import matplotlib.pyplot as plt
+from skada import KMMAdapter
+from skada.datasets import make_shifted_datasets
+
+# %%
+# Generate sample bias dataset
+# ----------------------------
+ds = make_shifted_datasets(
+        n_samples_source=12,
+        n_samples_target=3,
+        shift="covariate_shift",
+        label="binary",
+        noise=0.4,
+        random_state=123,
+        return_dataset=True
+)
+X, y, sample_domain = ds.pack_train(as_sources=['s'], as_targets=['t'])
+Xs, ys = ds.get_domain("s")
+Xt, yt = ds.get_domain("t")
+
+# %%
+# Illustration of Importance Weighting
```

Comment:
Very nice, maybe we could also illustrate the trained predictor with and woithout teh revwieghting? after tgis visualisation?